docs/products/postgresql/howto/migrate-db-to-aiven-via-console.rst

```diff
@@ -0,0 +1,308 @@
+Migrate PostgreSQL® databases to Aiven using the console 
+========================================================
+
+You can migrate PostgreSQL® databases to Aiven using either CLI (``aiven-db-migrate``) or the Aiven web console. This article addresses the latter scenario by providing guidelines on how to use the Aiven console to migrate PostgreSQL databases to the Aiven platform.
+
+.. seealso::
+
+    For the other migration method (using ``aiven-db-migrate``), see :doc:`Migrate to Aiven for PostgreSQL® with aiven-db-migrate </docs/products/postgresql/howto/migrate-aiven-db-migrate>`.
+
+About migrating via console
+---------------------------
+
+* Console migration tool enables you to migrate PostgreSQL databases to clusters in your Aiven account. You can migrate
+
+  * Existing on-premise PostgreSQL databases
+  * Cloud-hosted PostgreSQL databases
+  * Managed database clusters on Aiven.
+
+* Console migration tool uses `logical replication <https://www.postgresql.org/docs/current/logical-replication.html>`_ by default.
+
+* With the console migration tool, you can migrate PostgreSQL® databases either using the continuous migration method, which is the one recommended and detailed in this guide, or using the dump method (``pg_dump``).
+
+  * Continuous migration keeps the source database operational during the migration. It requires either superuser permissions or the ``aiven_extras`` extension.
+
+    * Superuser permissions can be verified using the guidelines in :ref:`Check out how to verify that you have superuser permissions <check-superuser>`. If you do not have superuser permissions, you can request them from your system administrator.
+    * ``aiven_extras`` extension (`GitHub repository <https://github.com/aiven/aiven-extras>`_) can be installed on your source database using the guidelines in :doc:`Install or update an extension </docs/products/postgresql/howto/manage-extensions>`.
+
+  * Dump is a point-in-time snapshot and requires the source database to be offline or in the maintenance mode during the migration process since any data it gets after initiating the dump are not migrated to the target database.
+
+.. _check-superuser:
+
+.. dropdown:: Expand to check out how to verify that you have superuser permissions.
+
+    Use ``psql`` to run the ``\du`` command:
+
+    .. code-block:: bash
+
+       \du
+
+    .. code-block:: bash
+       :caption: Expected output
+
+       Role name |                      Attributes                            |                 Member of
+       ----------+------------------------------------------------------------+-----------------------------------------
+       _dodb     | Superuser, Replication                                     | {}
```

Comment:
We should probably use a different `Role name` here, not sure DO appreciate us using their platform as a source of migration, suggesting users should migrate to aiven.